src/centroiders.cpp

```diff
@@ -27,7 +25,7 @@ std::vector<Star> DummyCentroidAlgorithm::Go(unsigned char *image, int imageWidt
     std::vector<Star> result;
 
     for (int i = 0; i < numStars; i++) {
-        result.push_back(Star(rand() % imageWidth, rand() % imageHeight, 10.0));
+        result.push_back(Star(rand() % imageWidth, rand() % imageHeight, 10.0)); // NOLINT
```

Comment:
just define a local variable to use as the seed for `rand_r`, set its initial value to whatever you want.